xml/Microsoft.Extensions.Hosting/HostBuilder.xml

```diff
@@ -104,9 +104,11 @@
       <Docs>
         <param name="configureDelegate">The delegate for configuring the <see cref="T:Microsoft.Extensions.Configuration.IConfigurationBuilder" /> that will be used
             to construct the <see cref="T:Microsoft.Extensions.Configuration.IConfiguration" /> for the application.</param>
-        <summary>Sets up the configuration for the remainder of the build process and application. This can be called multiple times and
-            the results will be additive. The results will be available at <see cref="P:Microsoft.Extensions.Hosting.HostBuilderContext.Configuration" /> for
-            subsequent operations, as well as in <see cref="P:Microsoft.Extensions.Hosting.IHost.Services" />.</summary>
+        <summary>Sets up the configuration for the remainder of the build process and application.
+            The <see cref="P:Microsoft.Extensions.Hosting.HostBuilderContext.Configuration"/> passed in is the host's configuration built
+            from calls to <see cref="M:Microsoft.Extensions.Hosting.IHostBuilder.ConfigureHostConfiguration(System.Action{Microsoft.Extensions.Configuration.IConfigurationBuilder})"/>. This can be called multiple times and the results will be additive.
+            After all calls have been processed, <see cref="P:Microsoft.Extensions.Hosting.HostBuilderContext.Configuration"/> will be updated with the results
+            for future build steps. The resulting configuration will also be available in the <see cref="P:Microsoft.Extensions.Hosting.IHost.Services"/> DI Container.</summary>
```

Comment:
@carlossanlop is it alright to break summary into multiple lines? just checking.

Reply:
While it's not forbidden, you have to keep in mind that this information is going to show up in Visual Studio via Intellisense when invoking this API. So if the text takes too many lines, the dialog may end up being too large.

When the text is too large for a _summary_, I normally suggest to split the text and put the less important information inside the remarks section.

@maryamariyan up to you to accept it as is, or suggest to put it all in one line, or to move some of the text into the remarks.